### PR TITLE
Return services from AddFluentValidation.

### PR DESCRIPTION
**Pull request by denis-ivanov**

### CODE REVIEW

File: src/MediatR.Extensions.FluentValidation.AspNetCore/MediatR.Extensions.FluentValidation.AspNetCore.csproj [hunk 1]
```diff
@@ -21,10 +21,10 @@
...
 
   <ItemGroup>
-    <PackageReference Include="FluentValidation" Version="8.2.2" />
-    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.2.2" />
-    <PackageReference Include="MediatR" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.0" />
+    <PackageReference Include="FluentValidation" Version="8.6.2" />
+    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.6.2" />
+    <PackageReference Include="MediatR" Version="8.0.1" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.5" />
```
Comment (GetoXs) [inline src/MediatR.Extensions.FluentValidation.AspNetCore/MediatR.Extensions.FluentValidation.AspNetCore.csproj]: Is it needed to change versions? It limit availability for others.

**Review by GetoXs — COMMENTED** [review]
CR

File: src/MediatR.Extensions.FluentValidation.AspNetCore/MediatR.Extensions.FluentValidation.AspNetCore.csproj (hunk 1 above)
Comment (denis-ivanov) [reply src/MediatR.Extensions.FluentValidation.AspNetCore/MediatR.Extensions.FluentValidation.AspNetCore.csproj]: Reverted

**Comment by GetoXs** [comment]
@denis-ivanov  FYI. published in new version https://www.nuget.org/packages/MediatR.Extensions.FluentValidation.AspNetCore/1.0.2

**Pull request merged**